server/routes/org_routes.py

```
@@ -1156,3 +1160,89 @@ def update_org_preferences(user_id):
+    if not org_id:
+        return jsonify({"error": "No organization found"}), 404
+
+    data = request.get_json() or {}
+    new_tier = data.get("plan_tier")
+    if new_tier not in ("free", "enterprise"):
+        return jsonify({"error": "plan_tier must be 'free' or 'enterprise'"}), 400
+
+    try:
+        removed_user_ids = []
+        with db_pool.get_admin_connection() as conn:
+            with conn.cursor() as cursor:
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)